tests/gate_data.py

```diff
@@ -93,6 +93,10 @@
 
 TCLOCK = np.array([[1, 0, 0], [0, OMEGA, 0], [0, 0, OMEGA**2]])  # Qutrit clock gate
 
+TH = (-1j / np.sqrt(3)) * np.array(
+    [[1, 1, 1], [1, OMEGA, OMEGA**2], [1, OMEGA**2, OMEGA]]
+)  # hadamard gate
+
```

Comment:
This Ternary Hadamard is the generalized Hadamard, which doesn't need a subspace for a complete definition. Any tests using the subspace variants of the Ternary Hadamard use manual definitions

Reply:
Would things be cleaner, say for future tests that need to be written with subspace variants of Ternary Hadamard? If so, might be worth adding, otherwise its fine to leave it as is.